stablehlo/dialect/TypeInference.h

```diff
@@ -163,6 +167,9 @@ LogicalResult inferSortOp(
     Region& comparator,
     SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes);
 
+LogicalResult verifySortOp(Optional<Location> location, ValueRange inputs,
+                           uint64_t dimension, Region& comparator);
+
 LogicalResult inferTransposeOp(Optional<Location> loc, Value operand,
                                DenseIntElementsAttr permutation,
                                SmallVectorImpl<Type>& inferredReturnTypes);
```

Comment:
Also, as discussed earlier today, can you update the PR to include all ops with regions which currently have type inference implemented for them?

Reply:
Sure, I also put the full list of ops with region in https://github.com/openxla/stablehlo/issues/623#issuecomment-1341682168